packages/shared/src/lib/constants.ts

```diff
@@ -132,16 +132,40 @@ export const feedRangeFilters: RadioItemProps[] = [
 
 export const customFeedsPlusDate = new Date('2024-12-11');
 
+/*
+  The list below must match the Confluence page attached
+  https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1571946510/Restricted+countries+for+business+activities+aka+sanctioned+countries#Restricted-Countries-and-Regions
+*/
 export const invalidPlusRegions = [
-  'CU',
-  'IR',
-  'MM',
-  'SD',
-  'SY',
-  'KP',
-  'BY',
-  'ZW',
-  'RU',
+  'AF', // Afghanistan
+  'AQ', // Antarctica
+  'BY', // Belarus
+  'MM', // Burma (Myanmar)
+  'CF', // Central African Republic
+  'UA-43', // Crimea (Region of Ukraine)
+  'CU', // Cuba
+  'CD', // Democratic Republic of Congo
+  'UA-14', // Donetsk (Region of Ukraine)
+  'HT', // Haiti
+  'IR', // Iran
+  'IQ', // Iraq
+  'UA-65', // Kherson (Region of Ukraine)
```

Comment:
Is this valid for google `x-client-region`, I looked before and it did not apply

Reply:
We can maybe check with Ido when he comes back on how we can allow this to work. Because if I'm not mistaken, this is also an ISO standard code for regions.

Reply:
You can merge as is, if it resolve it will be blocked, nothing much we can do about it